luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/factories/AbridgedLogicalTableGroupFactory.java

```diff
@@ -25,15 +25,21 @@
 
 /**
  * A factory that is used by default to support LogicalTableGroup.
+ *
+ * A LogicalTableGroup is essentially a map from TableIdentifier to LogicalTable.
+ * This concept exists for the reason that when we configure LogicalTable,
+ * each granularity will be treated as a separated LogicalTable internally.
+ * Thus one configuration correspond to a LogicalTableGroup, and the infrastructure
+ * will support look up for each individual LogicalTable with a TableIdentifier.
```

Comment:
Slight rewording of this:
"One configuration string (name) defines multiple LogicalTableIdentifiers (name, granularity pair).  LogicalTableGroup allows us to collect all the distinct LogicalTables for a single name, mapping LogicalTableIdentifier to LogicalTable.

At the external configuration layer, configuration users will define a logical table group (keyed by name) with a list of granularities.  The factory will produce the collection of LogicalTables keyed by identifier (name, grain pairs), to be merged into the table dictionary."

Reply:
changed to the doc you suggested.